python/packages/sdk/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### 0.1.1 (2023-02-22)
+
+### Features
+
+- Initial version ([3a8e22d](https://github.com/serverless/console/commit/3a8e22d38107d497a805584c2b49ab3a6026ba38))
```

Comment:
I think we should under v0.1.0 add comment: _Invalid PyPI release_, and at v0.1.1 state under "Maintenance improvements": _Fix PyPI release_